### ISSUE TITLE
Containerless Knockouts aren't evaluated inside sortable

### ISSUE THREAD

**Issue by sc-moonlight** [rniemeyer/knockout-sortable#147]
When I have a containerless if inside a sortable it is not evaluated ignored.  

I was able to create a simple example on jsfiddle to demonstrate
[https://jsfiddle.net/scfiddle/v6khuLkz/]

    
    <ul data-bind="sortable: buckets" class="buckets">
    
    <li data-bind="if: bool"></li>
    
    </ul>
    
    <ul data-bind="foreach: buckets">
    
    <li data-bind="if: bool"></li>
    
    </ul>

**Comment by rniemeyer** [comment]
@sc-moonlight - The sortable binding expects a single element inside the parent container and strips anything else, as it causes problems with the sorting logic. For now, you would have to pre-filter your list or hide things with visible binding.

**Comment by sc-moonlight** [comment]
OK.  So it's stripping any containerless binding right?  I tried a containerless template and that failed as well.  I rewrote my side to be a visible template binding, selecting the template based on an observable instead of using a ko:if.  

Yes, I ran into the single element issue, which is why I was trying to use a containerless binding to only have one element be there.  Could probably tweak the selector used to get the correct index  of the item to move, but the template works for me at this point
